docs/dev_guides/changing_splink/testing.md

```diff
@@ -30,6 +27,11 @@ or alternatively
 pytest tests/
 ```
 
+!!! information Note
+
+    If you do not have an AWS account, or have not set up Splink to use it, you will need to add
+    `--ignore tests/test_full_example_athena.py` to every `pytest` command you run.
```

Comment:
This is supposed to be handled automatically, but perhaps it's not working.

Specifically, [this](https://github.com/moj-analytical-services/splink/blob/bfaf8d142f7aad03f281b6324dd530e202eaec9a/tests/test_full_example_athena.py#L38) should cause the tests to be skipped if the test cannot create a valid connection to Athena